src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  --text-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  height: 60px;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #FFF;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header a {
+  width: 40px;
+  height: 40px;
+}
+
+.nav .nav__list {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  list-style: none;
+  position: relative;
+}
+
+.nav .nav__list .nav__item:not(:last-child) {
+  margin: 0 20px 0 0;
+}
+
+.nav .nav__list .nav__item a {
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  height: 60px;
+}
+
+.nav .nav__list .nav__item:hover{
+  color: var(--text-color);
+}
+
+.nav .nav__list .nav__item .is-active{
```

Comment:
Fix it everywhere
```suggestion
.is-active{
```

Reply:
Still not fixed from the previous review
<img width="827" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/f19ceb5c-dcad-4426-9878-8228f53197c0">